README.md

```
@@ -56,242 +66,248 @@ LLM（Ollama）
+* レイヤードアーキテクチャ + ユースケース駆動
+* フレームワーク非依存な Domain を中心に据えた設計
 
-Docker環境が既に起動していることを前提とします。Docker環境のセットアップについては、[habit_rpg_docker](https://github.com/naostudy0/habit_rpg_docker)リポジトリのREADMEを参照してください。
+## アーキテクチャ概要
 
-### 1. 依存関係のインストール
+### レイヤ構成
 
-```bash
-composer install
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)